database/insertData/07_review_assignments.js

```diff
@@ -18,7 +18,6 @@ exports.queries = [
             connectByUsername: { username: "testReviewer1" } 
           }
           status: ASSIGNED
-          templateSectionRestrictions: ["S1", "S2"]
```

Comment:
Don't you also need to put these back in `allowedSections` field?

Reply:
Nopes, the default is `null` and that means all-allowed. I will add to the documentation - forgot to do it

Reply:
Ah, got it :)